### DISCUSSION TITLE
system-clipboard being blocked by vimium extension

### DISCUSSION THREAD

**Discussion by bhavitsharma** [gitpod-io/openvscode-server#164] [Q&A] [answered]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.61.0
- OS Version: ubuntu:20.04

Steps to Reproduce:

1. Run the docker image of openvscode-server.
2. Install the vimium extension and copy paste anything from system's clipboard. 
3. You'll see an error 
```
Unable to read from the browser's clipboard. Please make sure you have granted access for this website to read from the clipboard.
```

**Comment by bhavitsharma** [comment]
Copying/Pasting works fine without the extension installed.

**Comment by axonasif** [comment]
How about you exclude your openvscode web address on vimium?
I do that for gitpod.io as well.

**Comment by mikenikles** [comment, marked as answer]
I recently documented that for Gitpod at https://www.gitpod.io/docs/configure/browser-settings and the same steps apply here - with a different domain wildcard 😉
